src/pages/api/auth/[...nextauth].js

```diff
@@ -32,6 +32,7 @@ if (process.env.KEYCLOAK_ID) {
       name: process.env.KEYCLOAK_NAME,
       clientSecret: process.env.KEYCLOAK_SECRET,
       issuer: process.env.KEYCLOAK_ISSUER,
+      allowDangerousEmailAccountLinking: process.env.KEYCLOAK_DANGER_EMAIL_ACC_LINK,
```

Comment:
Can we default this to false?

Reply:
If this variable not declared, wouldn't be it's value equal to zero, which means FALSE by boolean logic? But I'm not a developer and have no experience in nextjs...

Reply:
Yeah your right about the `undefined` being falsy, but just to be explicit about it 👍